src/layers/point-layer/point-layer.js

```diff
@@ -248,7 +248,7 @@ export default class PointLayer extends Layer {
         pickable: true,
         // parameters
         parameters: {
-          depthTest: mapState.dragRotate
+          depthTest: this.config.columns.altitude.fieldIdx > -1
```

Comment:
this should be, 
`mapState.dragRotate || this.config.columns.altitude.fieldIdx > -1`
We want to depth test in 3d mode so that the layer order is correctly shown

Reply:
I choose to go with ```this.config.columns.altitude.fieldIdx > -1``` only because this layer should be flat on the map when the altitude column is not used. It shouldn't matter whether in 3d mode or not.